# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Free Software Foundation, Inc.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: pybliographer-1.0.7\n"
"POT-Creation-Date: 2001-11-29 10:02+ZONE\n"
"PO-Revision-Date: 2003-08-05 21:20+0200\n"
"Last-Translator: Frédéric Gobry <gobry@users.sourceforge.net>\n"
"Language-Team: fr <fr@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: .././Pyblio/Autoload.py:57
msgid "warning: can't import %s: %s"
msgstr "attention: impossible d'importer %s : %s"

#: .././Pyblio/Config.py:53
msgid "value of `%s' should be of type %s"
msgstr "La valeur de `%s' devrait être de type %s"

#: .././Pyblio/Config.py:196
msgid "String"
msgstr "Chaîne"

#: .././Pyblio/Config.py:205
msgid "Boolean"
msgstr "Booléen"

#: .././Pyblio/Config.py:224
msgid "Integer"
msgstr "Entier"

#: .././Pyblio/Config.py:226
msgid "Integer under %d"
msgstr "Entier plus petit que %d"

#: .././Pyblio/Config.py:228
msgid "Integer over %d"
msgstr "Entier plus grand que %d"

#: .././Pyblio/Config.py:230
msgid "Integer between %d and %d"
msgstr "Entier entre %d et %d"

#: .././Pyblio/Config.py:242
msgid "Element in `%s'"
msgstr "Élément dans `%s'"

#: .././Pyblio/Config.py:262
msgid "Tuple (%s)"
msgstr "Séquence (%s)"

#: .././Pyblio/Config.py:283
msgid "List (%s)"
msgstr "Liste (%s)"

#: .././Pyblio/Config.py:308
msgid "Dictionary (%s, %s)"
msgstr "Dictionnaire (%s, %s)"

#: .././Pyblio/Fields.py:265
msgid "Illegal year value"
msgstr "Mauvaise valeur d'année"

#: .././Pyblio/Fields.py:269
msgid "Illegal month value"
msgstr "Mauvaise valeur de mois"

#: .././Pyblio/Fields.py:273
msgid "Illegal day value"
msgstr "Mauvaise valeur de jour"

#: .././Pyblio/Fields.py:439
msgid "in %s"
msgstr "dans %s"

#: .././Pyblio/LyX.py:39
msgid "no input pipe `%s'"
msgstr "pas de pipe d'entrée `%s'"

#: .././Pyblio/LyX.py:45
msgid "no output pipe `%s'"
msgstr "pas de pipe de sortie `%s'"

#: .././Pyblio/Open.py:97
msgid "method `%s' provides no opener"
msgstr "pas de fonction d'ouverture dans `%s'"

#: .././Pyblio/Open.py:105
msgid "File `%s' does not exist"
msgstr "Le fichier `%s' n'existe pas"

#: .././Pyblio/Open.py:116
msgid "don't know how to open `%s'"
msgstr "je ne sais pas comment ouvrir `%s'"

#: .././Pyblio/Open.py:143
msgid "method `%s' provides no iterator"
msgstr "pas d'itérateur dans la méthode `%s'"

#: .././Pyblio/Format/BibTeX.py:347
msgid "key `%s' is malformed"
msgstr "la clé `%s' est mal formée"

#: .././Pyblio/Format/BibTeX.py:368
msgid "%s:%d: key `%s' already defined"
msgstr "%s:%d: La clé `%s' existe déjà"

#: .././Pyblio/GnomeUI/Editor.py:38
msgid "Last Name, First Name"
msgstr "Nom de famille, Prénom"

#: .././Pyblio/GnomeUI/Editor.py:39
msgid "Text"
msgstr "Texte"

#: .././Pyblio/GnomeUI/Editor.py:42
msgid "Reference"
msgstr "Références..."

#: .././Pyblio/GnomeUI/Editor.py:258
msgid "Day"
msgstr "Jour"

#: .././Pyblio/GnomeUI/Editor.py:267
msgid "Month"
msgstr "Mois"

#: .././Pyblio/GnomeUI/Editor.py:276
msgid "Year"
msgstr "Année"

#: .././Pyblio/GnomeUI/Editor.py:301
msgid "Invalid day field in date"
msgstr "Le champ du jour est incorrect"

#: .././Pyblio/GnomeUI/Editor.py:309
msgid "Invalid month field in date"
msgstr "Le champ du mois est incorrect"

#: .././Pyblio/GnomeUI/Editor.py:317
msgid "Invalid year field in date"
msgstr "Le champ de l'année est incorrect"

#: .././Pyblio/GnomeUI/Editor.py:426
msgid "Entry type"
msgstr "Type d'entrée"

#: .././Pyblio/GnomeUI/Config.py:482 .././Pyblio/GnomeUI/Editor.py:428
msgid "Key"
msgstr "Clé"

#: .././Pyblio/GnomeUI/Editor.py:471
msgid "Create Field"
msgstr "Créer le champ"

#: .././Pyblio/GnomeUI/Editor.py:539 .././Pyblio/GnomeUI/Fields.py:191
msgid "Mandatory"
msgstr "Obligatoire"

#: .././Pyblio/GnomeUI/Editor.py:539 .././Pyblio/GnomeUI/Fields.py:192
msgid "Optional"
msgstr "Facultatif"

#: .././Pyblio/GnomeUI/Editor.py:539
msgid "Extra"
msgstr "Supplément"

#: .././Pyblio/GnomeUI/Editor.py:601
msgid "Invalid key format"
msgstr "Format de clé incorrecte"

#: .././Pyblio/GnomeUI/Editor.py:609
msgid "Key `%s' already exists"
msgstr "La clé `%s' existe déjà"

#: .././Pyblio/GnomeUI/Editor.py:676
msgid "Error in native string parsing"
msgstr "Erreur lors de l'analyse du texte natif"

#: .././Pyblio/GnomeUI/Editor.py:689
msgid "Edit entry"
msgstr "Édition d'une entrée"

#: .././Pyblio/GnomeUI/Editor.py:703
msgid "Native Editing"
msgstr "Édition native"

#: .././Pyblio/GnomeUI/Editor.py:756
msgid "Standard Editing"
msgstr "Édition standard"

#: .././Pyblio/GnomeUI/Document.py:196 .././Pyblio/GnomeUI/FileSelector.py:41
msgid "File"
msgstr "Fichier"

#: .././Pyblio/GnomeUI/FileSelector.py:74
msgid "Bibliography type:"
msgstr "Type de bibliographie :"

#: .././Pyblio/GnomeUI/Index.py:68
msgid "Add..."
msgstr "Ajouter..."

#: .././Pyblio/GnomeUI/Index.py:69
msgid "Edit..."
msgstr "Éditer..."

#: .././Pyblio/GnomeUI/Index.py:70 .././Pyblio/GnomeUI/Search.py:178
msgid "Delete"
msgstr "Effacer"

#: .././Pyblio/GnomeUI/Search.py:109
msgid "Search"
msgstr "Chercher"

#: .././Pyblio/GnomeUI/Fields.py:225 .././Pyblio/GnomeUI/Search.py:133
msgid "Field"
msgstr "Champ"

#: .././Pyblio/GnomeUI/Search.py:134
msgid "Pattern"
msgstr "Valeur"

#: .././Pyblio/GnomeUI/Search.py:151
msgid "Simple Search"
msgstr "Recherche simple"

#: .././Pyblio/GnomeUI/Search.py:155
msgid "Search command:"
msgstr "Commande de recherche :"

#: .././Pyblio/GnomeUI/Search.py:161
msgid "Expert Search"
msgstr "Recherche avancée"

#: .././Pyblio/GnomeUI/Search.py:199
msgid "Full database"
msgstr "Base de donnée complète"

#: .././Pyblio/GnomeUI/Search.py:241
msgid "internal error during evaluation"
msgstr "Erreur interne durant l'évaluation"

#: .././Pyblio/GnomeUI/Search.py:290
msgid ""
"while compiling %s\n"
"error: %s"
msgstr ""
"Pendant la compilation de `%s'\n"
"erreur : %s"

#: .././Pyblio/GnomeUI/Sort.py:42
msgid "Select sort criterions"
msgstr "Choisir les critères de tri"

#: .././Pyblio/GnomeUI/Sort.py:53
msgid "Sort direction"
msgstr "Sens du tri"

#: .././Pyblio/GnomeUI/Sort.py:53
msgid "Sort criterions"
msgstr "Critères de tri"

#: .././Pyblio/GnomeUI/Sort.py:66
msgid "[Entry Type]"
msgstr "[Type d'entrée]"

#: .././Pyblio/GnomeUI/Sort.py:67
msgid "[Key Value]"
msgstr "[Valeur de la Clé]"

#: .././Pyblio/GnomeUI/Sort.py:92
msgid "Reorder"
msgstr "Reclasser"

#: .././Pyblio/GnomeUI/Sort.py:93
msgid "Groups the active fields on the top of the list"
msgstr "Regroupe les champs actifs en haut de la list"

#: .././Pyblio/GnomeUI/Sort.py:97
msgid "Unselect all"
msgstr "Tout déselectionner"

#: .././Pyblio/GnomeUI/Sort.py:98
msgid "Removes all the sort criterions"
msgstr "Supprime tous les critères de tri"

#: .././Pyblio/GnomeUI/Utils.py:163
msgid ""
"The following errors occured:\n"
"\n"
msgstr ""
"Les erreurs suivantes se sont produites :\n"
"\n"

#: .././Pyblio/GnomeUI/Document.py:52
msgid "_New"
msgstr "_Nouveau..."

#: .././Pyblio/GnomeUI/Document.py:54
msgid "_Merge with..."
msgstr "_Fusionner avec..."

#: .././Pyblio/GnomeUI/Document.py:55
msgid "Medline Query..."
msgstr "Recherche Medline..."

#: .././Pyblio/GnomeUI/Document.py:59
msgid "_Previous Documents"
msgstr "_Documents précédents"

#: .././Pyblio/GnomeUI/Document.py:66
msgid "About..."
msgstr "À propos..."

#: .././Pyblio/GnomeUI/Document.py:67
msgid "Submit a Bug Report"
msgstr "Envoyer un rapport de Bug"

#: .././Pyblio/GnomeUI/Document.py:79
msgid "_Add..."
msgstr "_Ajouter..."

#: .././Pyblio/GnomeUI/Document.py:80
msgid "_Edit..."
msgstr "_Editer..."

#: .././Pyblio/GnomeUI/Document.py:81
msgid "_Delete"
msgstr "E_ffacer"

#: .././Pyblio/GnomeUI/Document.py:84
msgid "S_ort..."
msgstr "_Trier..."

#: .././Pyblio/GnomeUI/Document.py:88
msgid "_Cite"
msgstr "_Citer"

#: .././Pyblio/GnomeUI/Document.py:89
msgid "_Format..."
msgstr "_Formatter..."

#: .././Pyblio/GnomeUI/Document.py:93
msgid "_Fields..."
msgstr "_Champs..."

#: .././Pyblio/GnomeUI/Document.py:94
msgid "_Entries..."
msgstr "_Entrées..."

#: .././Pyblio/GnomeUI/Document.py:100
msgid "_File"
msgstr "_Fichier"

#: .././Pyblio/GnomeUI/Document.py:101
msgid "_Edit"
msgstr "_Editer"

#: .././Pyblio/GnomeUI/Document.py:103
msgid "_Settings"
msgstr "_Paramètres"

#: .././Pyblio/GnomeUI/Document.py:104
msgid "_Help"
msgstr "_Aide"

#: .././Pyblio/GnomeUI/Document.py:108
msgid "Open"
msgstr "Ouvrir"

#: .././Pyblio/GnomeUI/Document.py:109
msgid "Save"
msgstr "Sauver"

#: .././Pyblio/GnomeUI/Document.py:111
msgid "Find"
msgstr "Trouver"

#: .././Pyblio/GnomeUI/Document.py:112
msgid "Cite"
msgstr "Citer"

#: .././Pyblio/GnomeUI/Document.py:114
msgid "Close"
msgstr "Fermer"

#: .././Pyblio/GnomeUI/Document.py:196
msgid "Previous Documents"
msgstr "Documents précédents"

#: .././Pyblio/GnomeUI/Document.py:232
msgid ""
"can't open file `%s' for writing:\n"
"%s"
msgstr ""
"Impossible d'ouvrir `%s' en écriture :\n"
"%s"

#: .././Pyblio/GnomeUI/Document.py:253
msgid ""
"Error while parsing `%s':\n"
"%s"
msgstr ""
"Erreur pendant la lecture de `%s':\n"
"%s"

#: .././Pyblio/GnomeUI/Document.py:272
msgid "New database"
msgstr "Nouvelle base de données"

#: .././Pyblio/GnomeUI/Document.py:280
msgid "[no entry]"
msgstr "[pas d'entrée]"

#: .././Pyblio/GnomeUI/Document.py:281
msgid "[1 entry]"
msgstr "[1 entrée]"

#: .././Pyblio/GnomeUI/Document.py:282
msgid "[%d entries]"
msgstr "[%d entrées]"

#: .././Pyblio/GnomeUI/Document.py:285
msgid "[%d/1 entry]"
msgstr "[%d/1 entrée]"

#: .././Pyblio/GnomeUI/Document.py:286
msgid "[%d/%d entries]"
msgstr "[%d/%d entrées]"

#: .././Pyblio/GnomeUI/Document.py:291
msgid "[modified]"
msgstr "[modifiée]"

#: .././Pyblio/GnomeUI/Document.py:301
msgid ""
"The database has been modified.\n"
"Discard changes ?"
msgstr ""
"Cette base de données a été changée.\n"
"Abandonner les changements ?"

#: .././Pyblio/GnomeUI/Document.py:321
msgid "Enter your Medline query"
msgstr ""
"Entrez le terme\n"
"à chercher dans Medline"

#: .././Pyblio/GnomeUI/Document.py:327
msgid "Search string"
msgstr "Terme cherché"

#: .././Pyblio/GnomeUI/Document.py:331
msgid "Maximum number of results"
msgstr "Nombre maxi. de résultats"

#: .././Pyblio/GnomeUI/Document.py:351
msgid "Merge file"
msgstr "Fusionner le fichier"

#: .././Pyblio/GnomeUI/Document.py:363
msgid "Open error"
msgstr "Erreur à l'ouverture"

#: .././Pyblio/GnomeUI/Document.py:387
msgid "Merge status"
msgstr "Résultat de la fusion"

#: .././Pyblio/GnomeUI/Document.py:398
msgid "Open file"
msgstr "Ouvrir fichier"

#: .././Pyblio/GnomeUI/Document.py:450
msgid ""
"The database has been externally modified.\n"
"Overwrite changes ?"
msgstr ""
"Cette base de données a été changée en dehors du programme.\n"
"Écraser les changements ?"

#: .././Pyblio/GnomeUI/Document.py:460
msgid ""
"Unable to save `%s':\n"
"%s"
msgstr ""
"Impossible de sauver `%s' :\n"
"%s"

#: .././Pyblio/GnomeUI/Document.py:468
msgid ""
"An internal error occured during saving\n"
"Try to Save As..."
msgstr ""
"Une erreur interne s'est produite\n"
"Essayez d'Enregistrer Sous..."

#: .././Pyblio/GnomeUI/Document.py:482
msgid "Save As..."
msgstr "Enregistrer Sous..."

#: .././Pyblio/GnomeUI/Document.py:488
msgid ""
"The file `%s' already exists.\n"
"Overwrite it ?"
msgstr ""
"Le fichier `%s' existe.\n"
"Faut-il l'écraser ?"

#: .././Pyblio/GnomeUI/Document.py:495
msgid ""
"During opening:\n"
"%s"
msgstr ""
"Lors de l'ouverture :\n"
"%s"

#: .././Pyblio/GnomeUI/Document.py:513
msgid "Reopen error"
msgstr "Erreur à la réouverture"

#: .././Pyblio/GnomeUI/Document.py:554
msgid ""
"An entry called `%s' already exists.\n"
"Rename and add it anyway ?"
msgstr ""
"Une entrée appelée `%s' existe déjà.\n"
"Renommer et ajouter quand même ?"

#: .././Pyblio/GnomeUI/Document.py:591
msgid "Really remove all the entries ?"
msgstr "Voulez-vous supprimer toutes les entrées ?"

#: .././Pyblio/GnomeUI/Document.py:611
msgid "Create new entry"
msgstr "Créer une nouvelle entrée"

#: .././Pyblio/GnomeUI/Document.py:625
msgid "Really edit %d entries ?"
msgstr "Voulez-vous éditer %d entrées ?"

#: .././Pyblio/GnomeUI/Document.py:663
msgid "Remove all the %d entries ?"
msgstr "Supprimer toutes les %d entrées ?"

#: .././Pyblio/GnomeUI/Document.py:665
msgid "Remove entry `%s' ?"
msgstr "Voulez-vous supprimer l'entrée `%s' ?"

#: .././Pyblio/GnomeUI/Document.py:731
msgid ""
"Can't connect to LyX:\n"
"%s"
msgstr ""
"Impossible de se connecter à LyX :\n"
"%s"

#: .././Pyblio/GnomeUI/Document.py:785
msgid ""
"Please install bug-buddy\n"
"to use this feature"
msgstr ""
"Veuillez installer `bug-buddy'\n"
"pour utiliser cette fonction"

#: .././Pyblio/GnomeUI/Document.py:796
msgid "This program is copyrighted under the GNU GPL"
msgstr "Ce programme est placé sous la licence GNU GPL"

#: .././Pyblio/GnomeUI/Document.py:798
msgid "Gnome interface to the Pybliographer system."
msgstr "Une interface graphique pour Pybliographer"

#: .././Pyblio/GnomeUI/Document.py:803
msgid "Pybliographer Home Page"
msgstr "Page principale de Pybliographer"

#: .././Pyblio/GnomeUI/Format.py:37
msgid "Format entries"
msgstr "Formatter les entrées"

#: .././Pyblio/GnomeUI/Format.py:51
msgid "Bibliography style:"
msgstr "Type de bibliographie :"

#: .././Pyblio/GnomeUI/Format.py:52
msgid "Select style"
msgstr "Choix du style"

#: .././Pyblio/GnomeUI/Format.py:58
msgid "Output format:"
msgstr "Format de sortie :"

#: .././Pyblio/GnomeUI/Format.py:75
msgid "Output File:"
msgstr "Fichier de sortie :"

#: .././Pyblio/GnomeUI/Format.py:76
msgid "Select output file"
msgstr "Choisir le fichier de sortie"

#: .././Pyblio/GnomeUI/Config.py:45
msgid "Choose your preferences"
msgstr "Choisissez vos préférences"

#: .././Pyblio/GnomeUI/Config.py:93
msgid ""
"Item `%s':\n"
"\n"
"%s"
msgstr ""
"Élément `%s' :\n"
"\n"
"%s"

#: .././Pyblio/GnomeUI/Config.py:128 .././Pyblio/GnomeUI/Fields.py:157
msgid ""
"Some changes require to restart Pybliographic\n"
"to be correctly taken into account"
msgstr ""
"Certains changements nécessite de redémarrer Pybliographic\n"
"pour être correctement pris en compte"

#: .././Pyblio/GnomeUI/Config.py:249
msgid "True"
msgstr "Vrai"

#: .././Pyblio/GnomeUI/Config.py:250
msgid "False"
msgstr "Faux"

#: .././Pyblio/GnomeUI/Config.py:390
msgid "Add"
msgstr "Ajouter"

#: .././Pyblio/GnomeUI/Config.py:393
msgid "Update"
msgstr "Mettre à jour"

#: .././Pyblio/GnomeUI/Config.py:396
msgid "Remove"
msgstr "Supprimer"

#: .././Pyblio/GnomeUI/Config.py:482
msgid "Value"
msgstr "Valeur"

#: .././Pyblio/GnomeUI/Config.py:491 .././Pyblio/GnomeUI/Fields.py:93
msgid "Set"
msgstr "Appliquer"

#: .././Pyblio/GnomeUI/Config.py:507
msgid "Key:"
msgstr "Clé :"

#: .././Pyblio/GnomeUI/Config.py:509
msgid "Value:"
msgstr "Valeur :"

#: .././Pyblio/GnomeUI/Fields.py:45
msgid "Fields configuration"
msgstr "Configuration des Champs"

#: .././Pyblio/GnomeUI/Fields.py:59
msgid "Name"
msgstr "Nom"

#: .././Pyblio/GnomeUI/Fields.py:59
msgid "Type"
msgstr "Type"

#: .././Pyblio/GnomeUI/Fields.py:70
msgid "Name:"
msgstr "Nom :"

#: .././Pyblio/GnomeUI/Fields.py:72
msgid "Type:"
msgstr "Type :"

#: .././Pyblio/GnomeUI/Fields.py:198
msgid "Entries configuration"
msgstr "Configuration des Entrées"

#: .././Pyblio/GnomeUI/Fields.py:209
msgid "Entry"
msgstr "Entrée"

#: .././Pyblio/GnomeUI/Fields.py:220
msgid "Entry Name:"
msgstr "Nom de l'entrée :"

#: .././Pyblio/GnomeUI/Fields.py:225
msgid "Status"
msgstr "État"

#: .././pybliographer.py:48
msgid "This is %s, version %s"
msgstr "Bienvenue dans %s, version %s"

#: .././pybliographer.py:52
msgid "This is free software with ABSOLUTELY NO WARRANTY."
msgstr "Ceci est un logiciel libre, sans AUCUNE GARANTIE"

#: .././pybliographer.py:53
msgid "For details, type `warranty'."
msgstr "Pour plus de détails, tapez `warranty'."

#: .././pybliographer.py:116
msgid "%s: error: can't open file `%s'"
msgstr "%s: erreur : impossible d'ouvrir le fichier `%s'"

#: .././pybliographer.py:129
msgid "For help, run %s and type `help' at the prompt"
msgstr "Pour de l'aide, tapez %s puis le mot `help' à l'invite"

#: .././pybliographer.py:138
msgid ""
"Useful commands:\n"
"\thelp     to get some help\n"
"\tquit     to quit\n"
msgstr ""
"Commandes utiles :\n"
"\thelp\tpour obtenir de l'aide\n"
"\tquit\tpour quitter le programme\n"

#: .././scripts/pybcompact.py:29
msgid "usage: pybcompact <latexfile> <bibtexfiles...>"
msgstr "usage : pybcompact <fichier latex> <fichiers bibtex...>"

#: .././scripts/pybcompact.py:33
msgid "pybcompact: error: %s\n"
msgstr "pybcompact : erreur: %s\n"

#: .././scripts/pybcompact.py:96
msgid "no entry"
msgstr "pas d'entrée"

#: .././scripts/pybcompact.py:141
msgid "can't find the following entries: %s"
msgstr "impossible de trouver les entrées suivantes : %s"

#: .././scripts/pybconvert.py:28
msgid "usage: pybconvert <source>..<target> <input> [output]"
msgstr "usage : pybconvert <source>..<cible> <entrée> [<sortie>]"

#: .././scripts/pybconvert.py:37
msgid "pybconvert: bad conversion format"
msgstr "pybconvert : mauvaise spécification de format"

#: .././scripts/pybformat.py:33
msgid "usage: pybformat [bibtexfiles...]\n"
msgstr "usage : pybformat [fichiers bibtex...]\n"

#: .././scripts/pybformat.py:37
msgid "pybformat: error: %s\n"
msgstr "pybformat : erreur: %s\n"

#: .././scripts/pybformat.py:43
msgid "pybformat: warning: %s\n"
msgstr "pybformat : attention: %s\n"

#: .././scripts/pybformat.py:140 .././scripts/pybtex.py:85
msgid "can't find style `%s'"
msgstr "Impossible de trouver le style `%s'"

#: .././scripts/pybtex.py:33
msgid "usage: pybtex <latexfile> [bibtexfiles...]"
msgstr "usage : pybtex <fichier latex> [fichiers bibtex...]"

#: .././scripts/pybtex.py:37
msgid "pybtex: error: %s\n"
msgstr "pybtex : erreur: %s\n"

#: .././scripts/pybtex.py:58
msgid "pybtex: warning: the following keys were not resolved"
msgstr "pybtex : attention : les clés suivantes n'ont pas été trouvées"

#: .././scripts/pybtex.py:63
msgid "no style defined"
msgstr "pas de style défini"

#~ msgid "In use"
#~ msgstr "Utilisé"

#~ msgid "Yes"
#~ msgstr "Oui"

#~ msgid "New text"
#~ msgstr "Nouveau texte"

#~ msgid "an Instance of `%s'"
#~ msgstr "un Instance de `%s'"

#~ msgid "Error"
#~ msgstr "Erreur"

#~ msgid "Format"
#~ msgstr "Format"

#~ msgid "Formatting Style:"
#~ msgstr "Style de format:"

#~ msgid "entry has no key `%s'"
#~ msgstr "L'entrée n'a pas de clé `%s'"

#~ msgid "error: field `%s' appears more than once in the definition of `%s'"
#~ msgstr ""
#~ "erreur : le champ `%s' apparait plusieurs fois dans la définition de `%s'"

#~ msgid "database `%s' is unknown in this Reference"
#~ msgstr "la base de données `%s' est inconnue dans cette Reference"

#~ msgid "can't update a database of type Reference ()"
#~ msgstr "impossible de mettre à jour une base de données de type Reference"

#~ msgid "in entry %s: nested crossrefs"
#~ msgstr "dans l'entrée `%s' : références croisées imbriquées"

#~ msgid "Edit the field in its Native format"
#~ msgstr "Éditer le champ dans son format natif"

#~ msgid "Drag an Entry to define a Cross-Reference"
#~ msgstr "Déplacer une Entrée ici pour définir une référence croisée"

#~ msgid "Remove the current Cross-Reference"
#~ msgstr "Supprimer la référence croisée"

#~ msgid "Add Extra Field"
#~ msgstr "Ajouter un champ supplémentaire"

#~ msgid "Cross-Reference `%s' is unexistent in the database"
#~ msgstr "La référence croisée `%s' n'existe pas dans cette base"

#~ msgid "Invalid identifier `%s'"
#~ msgstr "L'identificateur `%s' est incorrect"

#~ msgid "Identifier `%s' already exists"
#~ msgstr "L'identificateur `%s' existe déjà"

#~ msgid "Quit"
#~ msgstr "Quitter"

#~ msgid "Copy"
#~ msgstr "Copier"

#~ msgid "Cut"
#~ msgstr "Couper"

#~ msgid "Paste"
#~ msgstr "Valeur"

#~ msgid "New..."
#~ msgstr "Nouveau..."

#~ msgid "Search..."
#~ msgstr "Chercher"

#~ msgid "Sort by Key"
#~ msgstr "Trier par Clé"

#~ msgid "Sort by Type"
#~ msgstr "Trier par Type"

#~ msgid "Cite in LyX"
#~ msgstr "Citer dans LyX"

#~ msgid "Configure"
#~ msgstr "Configurer"

#~ msgid "Exit"
#~ msgstr "Quitter"

#~ msgid "Welcome to Pybliographic"
#~ msgstr "Bienvenue dans Pybliographic"

#~ msgid "Modified database `%s' [%s, %s]"
#~ msgstr "Base de données modifiée `%s' [%s, %s]"

#~ msgid "Database `%s' [%s, %s]"
#~ msgstr "Base de données `%s' [%s, %s]"

#~ msgid "Database is modified.\n"
#~ msgstr "Base de données modifiée\n"

#~ msgid "Discard changes ?"
#~ msgstr "Oublier les modifications ?"

#~ msgid "Quit without saving ?"
#~ msgstr "Quitter sans sauver ?"

#~ msgid "Save As"
#~ msgstr "Sauver comme"

#~ msgid ""
#~ "Can't save file:\n"
#~ "%s"
#~ msgstr ""
#~ "Impossible de sauver le fichier:\n"
#~ "%s"

#~ msgid "Not yet implemented..."
#~ msgstr "Pas encore implémenté..."

#~ msgid ""
#~ "Please, create a new database first.\n"
#~ "Use File/New for example"
#~ msgstr ""
#~ "Veuillez créer une nouvelle base avant tout.\n"
#~ "Utilisez le menu Fichier/Nouveau pour cela"

#~ msgid "editing is disabled in this database"
#~ msgstr "L'édition n'est pas autorisée dans cette base"

#~ msgid ""
#~ "Submit a bug report\n"
#~ " using bug-buddy"
#~ msgstr ""
#~ "Envoyer un rapport de Bug\n"
#~ "en utilisant `bug-buddy'"

#~ msgid "Please open a database first"
#~ msgstr "Veuillez ouvrir une base de données"

#~ msgid "Bibliography"
#~ msgstr "Bibliographie"

#~ msgid "Ready to search !"
#~ msgstr "Prêt à chercher !"

#~ msgid "found %d matches"
#~ msgstr "%d entrées trouvées"

#~ msgid "%d matches in this node"
#~ msgstr "%d entrées dans ce noeud"

#~ msgid "edition is disabled in this database"
#~ msgstr "L'édition n'est pas autorisée dans cette base"

#~ msgid "Cross-Reference `%s' is unexistant in the database"
#~ msgstr "La référence croisée `%s' n'existe pas dans cette base"
